tools/devops/automation/templates/main-stage.yml

```diff
@@ -73,28 +73,123 @@ parameters:
 - name: macOSName
   type: string
 
-- name: simTestsConfigurations
+# Ideally we should read/get the list of platforms from somewhere else, instead of hardcoding them here.
+# Note that this is _all_ the platforms we support (not just the enabled ones).
+- name: supportedPlatforms
   type: object
   default: [
-    'bcl',
-    'cecil',
-    'dotnettests',
-    'fsharp',
-    'framework',
-    'generator',
-    'interdependent-binding-projects',
-    'install-source',
-    'introspection',
-    'linker',
-    'mac-binding-project',
-    'mmp',
-    'mononative',
-    'monotouch',
-    'msbuild',
-    'mtouch',
-    'xammac',
-    'xcframework',
-    'xtro'
+    {
+      platform: iOS,
+      isDotNetPlatform: true,
+      isLegacyPlatform: true
+    },
+    {
+      platform: macOS,
+      isDotNetPlatform: true,
+      isLegacyPlatform: true
+    },
+    {
+      platform: tvOS,
+      isDotNetPlatform: true,
+      isLegacyPlatform: true
+    },
+    {
+      platform: watchOS,
+      isDotNetPlatform: false,
+      isLegacyPlatform: true
+    },
+    {
+      platform: MacCatalyst,
+      isDotNetPlatform: true,
+      isLegacyPlatform: false
+    },
+    {
+      # when running platform-specific test runs, we also need a special test run that executes tests that only runs when multiple platforms are enabled
+      platform: Multiple,
+      isDotNetPlatform: true,
+      isLegacyPlatform: true
+    }
+  ]
+
+- name: testConfigurations
+  type: object
+  default: [
+    # {
+    #   label: bcl,
+    #   splitByPlatforms: false,
+    # },
+    {
+      label: cecil,
+      splitByPlatforms: false,
+      containsDotNetTests: true,
+      containsLegacyTests: true,
+    },
+    {
+      label: dotnettests,
+      splitByPlatforms: true,
+      containsDotNetTests: true,
+      containsLegacyTests: false,
+    },
+    # {
```

Comment:
This should be uncommented :)

Reply:
I was wondering where those tests runs were...